docs/pages/get-started/installation.md

```diff
@@ -4,50 +4,64 @@ title: Installation
 
 import { Terminal } from '~/ui/components/Snippet';
 
-There are two tools that you need to develop apps with Expo: a command line app called Expo CLI to initialize and serve your project and a mobile client app called Expo Go to open it on iOS and Android. Any web browser will work for opening the project on the web.
+To develop applications with Expo, you need two tools. A command-line application called [Expo CLI](#1-expo-cli) to initialize and serve your project, and a mobile client app called [Expo Go](#2-expo-go-app-for-ios-and) to open the project on iOS and Android platforms. You can use any web browser to run the project on the web.
 
-> 💡 You don't need macOS to build an iOS app with Expo, you only need an iOS device to run the Expo Go app. For your development machine, Windows, Linux, and macOS are all supported.
+> You don't need macOS to build an iOS app with Expo. You only need an iOS device to run the Expo Go app. Windows, Linux, and macOS are all supported for your development machine.
 
 ## 1. Expo CLI
 
-Expo CLI is a command line app that is the main interface between a developer and Expo tools. Expo CLI also has a web-based GUI that pops up in your web browser when you start your project &mdash; you can use the GUI instead of the command line interface if you're not yet comfortable using a terminal or prefer GUIs, both have similar capabilities.
+[Expo CLI](../workflow/expo-cli.md) is a command-line app that is the primary interface between a developer and Expo tools. It has a web-based Graphical User Interface (GUI) that pops up in the web browser when you start your project. You can use the GUI instead of the command line interface if you are not yet comfortable using a terminal or prefer GUIs. Both have similar capabilities.
 
 ### Requirements
 
+With Expo CLI, you need to have the following tools installed on your developer machine:
+
 - [Node.js LTS release](https://nodejs.org/en/)
 - [Git](https://git-scm.com)
-- [Watchman](https://facebook.github.io/watchman/docs/install#buildinstall) for macOS or Linux users
+- [Watchman](https://facebook.github.io/watchman/docs/install#buildinstall) required only for macOS or Linux users
 
-> Only Node.js LTS releases (even-numbered) are recommended. As Node.js [officially states](https://nodejs.org/en/about/releases/), "Production applications should only use Active LTS or Maintenance LTS releases."
+> Only Node.js LTS releases (even-numbered) are recommended. As Node.js [officially states](https://nodejs.org/en/about/releases/), "Production applications should only use Active LTS, or Maintenance LTS releases."
```

Comment:
Don't add the comma, it's a quote, and their page doesn't have it.

Reply:
Okay, will revert this. Thanks!